### ISSUE TITLE
[BUG] export does not respect current theme

### ISSUE THREAD

**Issue by HolleJ** [bokeh/bokeh#10804]
#### ALL software version info
bokeh: 2.2.3
Python: 3.8.6
OS: Windows 10
Browser: Chrome 87.0.4280.88

#### Description of expected behavior and the observed behavior
The theme of the the current document is not considered, if a svg export is used.

Part of expected result via save() (or show())
![image](https://user-images.githubusercontent.com/36547941/103346969-e2c3d700-4a95-11eb-8d27-454883a29bba.png)
Part of unexpected result via export_svgs()
![image](https://user-images.githubusercontent.com/36547941/103346985-eb1c1200-4a95-11eb-86ac-308968393dc7.png)

#### Complete, minimal, self-contained example code that reproduces the issue

```python
from bokeh.io import curdoc, export, save
from bokeh.plotting import figure
from bokeh.themes import Theme
from selenium import webdriver

CHROMEDRIVER_PATH = "chromedriver.exe"

curdoc().theme = Theme(json={'attrs': {
    'Figure': {
        'background_fill_color': '#2F2F2F'
    }
}})

p = figure()
p.line(x=[0, 1], y=[0, 1])

p.output_backend = "svg"
export.export_svgs(p, filename="export.svg", webdriver=webdriver.Chrome(CHROMEDRIVER_PATH))
save(p, filename="export.html")
```

#### Possible bug location / solution

According to the call of file_html
https://github.com/bokeh/bokeh/blob/d7a31e9a9604661bee0912633615c332732079ef/bokeh/io/saving.py#L148
triggered by
https://github.com/bokeh/bokeh/blob/d7a31e9a9604661bee0912633615c332732079ef/bokeh/io/saving.py#L80-L86

I assume that in get_layout_html (used by all export functions) the call
https://github.com/bokeh/bokeh/blob/d7a31e9a9604661bee0912633615c332732079ef/bokeh/io/export.py#L304-L305
should be extended:
```python
    from .state import curstate
    try:
        html = file_html(obj, resources, title="", template=template, suppress_callback_warning=True, _always_new=True, theme=curstate().document.theme)
```

If this is right, I could try to create a pull request - either as shown above (with import moved to the top) or with an additional param theme on all export methods to allow manual setting of the theme. Or did I miss something and the missing theme param was by intention?

**Comment by bryevdv** [comment]
@HolleJ Thanks for the detailed issue. I would say go ahead and submit a PR so that things can be easier to concretely test and discuss. That may be the solution but we will need to go back to some other recent theming issues and make sure to test that this fix doesn't undo other fixes (and also figure out a good way to maintain things under test)

**Comment by bryevdv** [comment]
potentially related to https://github.com/bokeh/bokeh/issues/10268

**Comment by bryevdv** [comment]
ref: comment about a census of current capability and problems: https://github.com/bokeh/bokeh/pull/10805#issuecomment-753527041

**Comment by bryevdv** [comment]
Changing title as this affects PNG export as well (I believe this was previously known in the PR comment) An MRE for PNG export can be found here: https://discourse.bokeh.org/t/theme-for-both-show-and-export-png/8774

**Comment by bryevdv** [comment]
We should probably try to fix this for 3.0

**Comment by shr00mie** [comment]
> We should probably try to fix this for 3.0

oh good. you guys are already on it. super. carry on.

**Comment by allefeld** [comment]
Is there a workaround to get theme-based SVGs without waiting for 3.0?
Or, for when is 3.0 planned?
